R/dataframe__frame.R

```diff
@@ -969,163 +1063,197 @@ DataFrame_to_struct = function(name = "") {
-#' @return A new `DataFrame` object with applied aggregation.
+#' @param ddof Delta Degrees of Freedom: the divisor used in the calculation is
+#' N - ddof, where N represents the number of elements. By default ddof is 1.
+#' @return A DataFrame with one row.
 #' @examples pl$DataFrame(mtcars)$var()
+
 DataFrame_var = function(ddof = 1) {
   self$lazy()$var(ddof)$collect()
 }
 
 #' @title Std
-#' @description Aggregate the columns of this DataFrame to their standard deviation values.
+#' @description Aggregate the columns of this DataFrame to their standard
+#' deviation values.
 #' @keywords DataFrame
-#' @param ddof integer Delta Degrees of Freedom: the divisor used in the calculation is N - ddof, where N represents the number of elements. By default ddof is 1.
-#' @return A new `DataFrame` object with applied aggregation.
+#' @param ddof Delta Degrees of Freedom: the divisor used in the calculation is
+#' N - ddof, where N represents the number of elements. By default ddof is 1.
+#' @return A DataFrame with one row.
 #' @examples pl$DataFrame(mtcars)$std()
+
 DataFrame_std = function(ddof = 1) {
   self$lazy()$std(ddof)$collect()
 }
 
 #' @title Quantile
-#' @description Aggregate the columns in the DataFrame to their quantile value.
+#' @description Aggregate the columns in the DataFrame to a unique quantile
+#' value. Use `$describe()` to specify several quantiles.
 #' @keywords DataFrame
-#' @param quantile numeric Quantile between 0.0 and 1.0.
-#' @param interpolation string Interpolation method: "nearest", "higher", "lower", "midpoint", or "linear".
+#' @param quantile Numeric of length 1 between 0 and 1.
+#' @param interpolation Interpolation method: "nearest", "higher", "lower",
+#' "midpoint", or "linear".
 #' @return DataFrame
 #' @examples pl$DataFrame(mtcars)$quantile(.4)
+
 DataFrame_quantile = function(quantile, interpolation = "nearest") {
   self$lazy()$quantile(quantile, interpolation)$collect()
 }
 
 #' @title Reverse
-#' @description Reverse the DataFrame.
-#' @keywords LazyFrame
+#' @description Reverse the DataFrame (the last row becomes the first one, etc.).
 #' @return DataFrame
 #' @examples pl$DataFrame(mtcars)$reverse()
+
 DataFrame_reverse = function() {
   self$lazy()$reverse()$collect()
 }
 
-#' @title Fill NaN
-#' @description Fill floating point NaN values by an Expression evaluation.
+#' @title Fill `NaN`
+#' @description Fill `NaN` values by an Expression evaluation.
 #' @keywords DataFrame
-#' @param fill_value Value to fill NaN with.
+#' @param fill_value Value to fill `NaN` with.
 #' @return DataFrame
 #' @examples
 #' df = pl$DataFrame(
 #'   a = c(1.5, 2, NaN, 4),
 #'   b = c(1.5, NaN, NaN, 4)
 #' )
 #' df$fill_nan(99)
+
 DataFrame_fill_nan = function(fill_value) {
   self$lazy()$fill_nan(fill_value)$collect()
 }
 
-#' @title Fill null
-#' @description Fill null values using the specified value or strategy.
+#' @title Fill nulls
+#' @description Fill null values (which correspond to `NA` in a classic
```

Comment:
I don't think the word "classic" is appropriate.
Isn't it just `NA` in R and `null` in Polars?

Reply:
Right, I replaced "in a classic data.frame" by "in R"